src/helpers/genesisProtocolHelper.js

```diff
@@ -0,0 +1,39 @@
+const parameterNames = [
+  'preBoostedVoteRequiredPercentage',
+  'preBoostedVotePeriodLimit',
+  'boostedVotePeriodLimit',
+  'thresholdConstA',
+  'thresholdConstB',
+  'minimumStakingFee',
+  'quietEndingPeriod',
+  'proposingRepRewardConstA',
+  'proposingRepRewardConstB',
+  'stakerFeeRatioForVoters',
+  'votersReputationLossRatio',
+  'votersGainRepRatioFromLostRep',
+  'daoBountyConst',
+  'daoBountyLimit'
+]
+
+const genesisProtocolConf = require('../config/votingMachines/GenesisProtocol')
+const { voteOnBehalf } = genesisProtocolConf
+
+const parameters = parameterNames.map(parameterName => genesisProtocolConf[parameterName])
+
+module.exports = artifacts => {
+  const DxGenesisProtocol = artifacts.require('DxGenesisProtocol')
+
+  async function getGenesisProtocolData () {
+    const dxGenesisProtocol = await DxGenesisProtocol.deployed()
```

Comment:
Ok, I see. In another PR I said the we shouldn't get this from config, we should use the `.deployed()` fn, I see u were already doing it, it's just that u created this helper.

Nice!